backend/src/graphql.ts

```diff
@@ -191,7 +190,42 @@ const Mutation = mutationType({
         })
         return user
       },
-    })
+    }),
+      t.field('loginUser', {
```

Comment:
I don't know if we have tabs mixed in on this file, but the indentation always looks different (and wrong) on github.

Reply:
Yeah I've noticed that, too. Almost mentioned it earlier!